python/source/CMakeLists.txt

```diff
@@ -21,7 +21,11 @@ SET_TARGET_PROPERTIES(pycap PROPERTIES
     SUFFIX ".so"
     )
 
-TARGET_LINK_LIBRARIES(pycap ${Boost_LIBRARIES} ${PYTHON_LIBRARIES} cap)
+TARGET_LINK_LIBRARIES(pycap cap)
+TARGET_LINK_LIBRARIES(pycap ${Boost_MPI_LIBRARY})
+TARGET_LINK_LIBRARIES(pycap ${Boost_SERIALIZATION_LIBRARY})
+TARGET_LINK_LIBRARIES(pycap ${Boost_PYTHON_LIBRARY})
+TARGET_LINK_LIBRARIES(pycap ${PYTHON_LIBRARIES})
```

Comment:
Are you sure you didn't forget some libraries? I had this error before:
```ImportError: libboost_timer.so.1.60.0: cannot open shared object file: No such file or directory```
 I fixed it by exporting the whole boost library